packages/mdc-chips/README.md

```diff
@@ -209,24 +209,27 @@ To use the `MDCChip` and `MDCChipSet` classes, [import](../../docs/importing-js.
 
 #### `MDCChip`
 
-Method Signature | Description
---- | ---
-`isSelected() => boolean` | Proxies to the foundation's `isSelected` method
-`beginExit() => void` | Proxies to the foundation's `beginExit` method
-
 Property | Value Type | Description
 --- | --- | ---
-`foundation` | MDCChipFoundation | The foundation
-`shouldRemoveOnTrailingIconClick` | Boolean | Proxies to the foundation's `getShouldRemoveOnTrailingIconClick`/`setShouldRemoveOnTrailingIconClick` methods
+`id` | string | Unique identifier on the chip*
+`selected` | Boolean | Proxies to the foundation's `isSelected`/`setSelected` methods
+`shouldRemoveOnTrailingIconClick` | Boolean | Proxies to the foundation's `getShouldRemoveOnTrailingIconClick`/`setShouldRemoveOnTrailingIconClick` methods**
 `ripple` | `MDCRipple` | The `MDCRipple` instance for the root element that `MDCChip` initializes
 
->_NOTE_: If `shouldRemoveOnTrailingIconClick` is set to false, you must manually call `beginExit()` on the chip to remove it.
+> *_NOTE_: By default, this will be the same as the `id` attribute on the root element. If an `id` is not provided, a random one will be generated.
```

Comment:
I would suggest escaping the `*`s (`\*`) in front of these notes and in the tables to avoid formatting bugs. It doesn't appear to cause any right now, but `*` is normally interchangeable with `_` in markdown (and they're currently fooling github's diff view as well as Sublime Text's markdown syntax highlighting).